doxygen/src/devices_schema.txt

```diff
@@ -2370,6 +2370,10 @@ These values can be used in the elements:
     <td class="XML-Token">Cortex-M55</td>
     <td><a href="http://developer.arm.com/ip-products/processors/cortex-m/cortex-m55" target="_blank">Arm Cortex-M55</a> processor based device</td>
   </tr>  
+  <tr>
+    <td class="XML-Token">Star</td>
+    <td><a href="https://www.armchina.com/mountain?infoId=160" target="_blank">ArmChina STAR-MC1</a> processor based device</td>
```

Comment:
I wish we had a url for a web page in english but I could not find one either.

Reply:
There seems to be non, unfortunately.